src/model.js

```diff
@@ -424,11 +434,24 @@ const BookshelfModel = ModelBase.extend({
     let columnNames, candidates;
     if (_.isArray(arguments[1])) {
       columnNames = arguments[1];
-      candidates = _.drop(arguments, 2);
     } else {
       columnNames = null;
-      candidates = _.drop(arguments);
     }
```

Comment:
In original API the second parameter as [`columnNames` is optional](http://bookshelfjs.org/#Model-instance-morphTo). Previously the given parameter was the only type of array. Now as you are using arrays to declare `morphValue` to you can not make `columnNames` optional.